Library/Formula/puddletag.rb

```diff
@@ -55,6 +49,9 @@ def install
 
     bin.install Dir[libexec/"bin/*"]
     bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+
+    system "sh create_macos_app_bundle.sh --name Puddletag --icon puddletag.png --script #{bin}/puddletag"
```

Comment:
In the future I try to stick with the 80 characters.
The linkapps thing is, well, stupid one the one hand, nice on the other... Being lazy I leave it now this way and try to avoid it in the future (or when linkapps is kicked on day, I'll bury it...)

Thanks a lot!